SUPPORT.md

```diff
@@ -13,18 +13,18 @@ Here are some good entry points to get started with our documentation including
 
 Project   | Links
 --------- | ----------------------------------------
-Flux (v2) | [Entry point](https://toolkit.fluxcd.io/), [Core Concepts](https://toolkit.fluxcd.io/core-concepts/), [Getting Started Guide](https://toolkit.fluxcd.io/get-started/), [FAQ](https://toolkit.fluxcd.io/faq/)
-Flagger   | [Entry point](https://docs.flagger.app/), [Getting Started Guide](https://docs.flagger.app/install/flagger-install-on-kubernetes), [FAQ](https://docs.flagger.app/faq)
-Helm Operator (v1) | [Entry point](https://docs.fluxcd.io/projects/helm-operator/en/stable/), [Getting Started Guide](https://docs.fluxcd.io/projects/helm-operator/en/stable/get-started/quickstart/), [FAQ](https://docs.fluxcd.io/projects/helm-operator/en/stable/faq/), [Troubleshooting](https://docs.fluxcd.io/projects/helm-operator/en/stable/troubleshooting/)
-Flux (v1 - in maintenance mode) | [Entry point](https://docs.fluxcd.io/en/stable/), [Getting Started Guide](https://docs.fluxcd.io/en/stable/get-started/), [FAQ](https://docs.fluxcd.io/en/stable/faq/), [Troubleshooting](https://docs.fluxcd.io/en/stable/troubleshooting/)
+Flux | [Entry point](https://fluxcd.io/docs/), [Core Concepts](https://fluxcd.io/docs/concepts/), [Getting Started Guide](https://fluxcd.io/docs/get-started/), [FAQ](https://fluxcd.io/docs/faq/)
+Flagger | [Entry point](https://docs.flagger.app/), [Getting Started Guide](https://docs.flagger.app/install/flagger-install-on-kubernetes), [FAQ](https://docs.flagger.app/faq)
+Helm Operator (legacy - v1) | [Entry point](https://fluxcd.io/legacy/helm-operator/), [Getting Started Guide](https://fluxcd.io/legacy/helm-operator/get-started/quickstart/), [FAQ](https://fluxcd.io/legacy/helm-operator/faq/), [Troubleshooting](https://fluxcd.io/legacy/helm-operator/troubleshooting/)
+Flux (legacy - v1 - in maintenance mode) | [Entry point](https://fluxcd.io/legacy/flux/), [Getting Started Guide](https://fluxcd.io/legacy/flux/get-started/), [FAQ](https://fluxcd.io/legacy/flux/faq/), [Troubleshooting](https://fluxcd.io/legacy/flux/troubleshooting/)
```

Comment:
I would remove `in maintenance mode` from here or add it to Helm Operator too.

Reply:
:+1: